src/plugins/index_pattern_field_editor/public/components/preview/field_preview.tsx

```diff
@@ -56,7 +56,7 @@ export const FieldPreview = () => {
     const [field] = fields;
 
     return (
-      <ul>
+      <ul tabIndex={0}>
```

Comment:
I've added this `tabIndex` so the list of field(s) to preview (which is the main focus point of the panel) is accessible through tab and accessible through voice over.

<img width="439" alt="Screenshot 2021-08-09 at 09 36 27" src="https://user-images.githubusercontent.com/2854616/128673233-27798845-6a31-443a-ab1e-4beb74acf13a.png">

Reply:
If there's no action to take here, we don't need it to be focusable so I think you can remove the `tabIndex`

Reply:
Would you mind adding comments to the code, so someone who's reading it in the future will understand the intention? Same request applies to the other use of `tabIndex={0}`.

```jsx
{/* Explanatory comment here */}
<ul tabIndex={0}>
```